MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -305,7 +306,7 @@ private void calcMedicalSupportHoursNeeded() {
         int activePersonnelCount = getCampaign().getActivePersonnel().size();
 
         // Calculated based on 7-person squads
-        int numSquads = (int) (activePersonnelCount / 7); // integer division intended
```

Comment:
The cast here was to ensure static analysis tools knew we wanted integer division.

Reply:
Idea is detecting this fine, and it looks like LGTM is too, which makes the cast redundant (the comment is super useful if you look there though)